src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.model.AreaCalculator;
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class HelloWorld {
+    public static void main(String[] args) {
+
+        System.out.print(String.format("%s. Color: %s. ", this.getClass().getSimpleName(), color));
```

Comment:
The variable `color` is used but not defined in this context. Ensure that `color` is properly defined and accessible.